.travis.yml

```diff
@@ -0,0 +1,84 @@
+sudo: required
+dist: trusty
+
+os: linux
+
+language: c
+
+compiler:
+  - gcc
+#  - clang
```

Comment:
@chipitsine Okay, I see. Thanks! I'll merge your current PR first. Feel free to create new PRs to address these little things :)